### PR TITLE
Initializing multiple blocks

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Fixed the bug in which block labels where not initialized when adding multiple blocks at once.

### CODE REVIEW

**Comment by martijn9612** [comment]
Looks good, glad you fixed the bug!

File: src/main/java/control/ControllerManager.java [hunk 1]
```diff
@@ -30,11 +31,13 @@
...
 
     // Placeholder project in lieu of XML loading
-    @Getter @Setter
+    @Getter
+    @Setter
     private ScriptingProject scriptingProject = new ScriptingProject("BOSS Project", 1.0);
 
     /**
      * Constructor.
+     *
```
Comment (Markvanderuit) [inline src/main/java/control/ControllerManager.java:40]: whitespace

File: src/main/java/control/ToolViewController.java [hunk 2]
```diff
@@ -68,12 +68,11 @@ private void createDirectorShot(DirectorShotCreationEvent event) {
...
                                              (int) event.getShotStart(), (int) event.getShotEnd());
 
-        TimelineController tlineControl = this.controllerManager.getTimelineControl();
-        event.getCamerasInShot().forEach(camInd -> {
-                tlineControl.addCameraShot(camInd,
-                                           event.getShotName(), event.getShotDescription(),
-                                           (int) event.getShotStart(), (int) event.getShotEnd());
-            });
+        TimelineController timelineController = this.controllerManager.getTimelineControl();
+        event.getCamerasInShot().forEach(camInd ->
```
Comment (martijnjanssen) [inline src/main/java/control/ToolViewController.java]: Please encapsulate the body in parentheses.

File: src/main/java/control/ControllerManager.java (hunk 1 above)
Comment (bart-de-jonge) [inline src/main/java/control/ControllerManager.java:40]: This is on purpose for readability of the comments. It doesn't add much in this case but I want to be consistent

**Comment by Markvanderuit** [comment]
Looks fine, other member can merge.

**Pull request merged**